hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncAdmin.java

```diff
@@ -1506,5 +1508,63 @@
    *   The return value will be wrapped by a {@link CompletableFuture}.
    */
   CompletableFuture<Boolean> isSnapshotCleanupEnabled();
+  
+  /**
+   * Gets group info for the given group name
+   * @param groupName the group name
+   * @return group info
+   */
+  CompletableFuture<? extends RSGroupInfo> getRSGroupInfo(String groupName);
```

Comment:
Why extends? IIRC we do not have any sub classes which extend RSGroupInfo?

Reply:
Yes, I'll change it.